### ISSUE TITLE
Where to start with accessibility?

### ISSUE THREAD

**Issue by HugoGiraudel** [HugoGiraudel/ama#87]
*Note: this question has been asked by [Viktor Leviné](https://twitter.com/viktorlevine_) on Twitter who kindly accepted I repost it here.*

Hi Hugo!

I’ve been looking to dive in to the world of accessibility to be able to create a more complete experience for everyone that visit my works. But I don’t know where to begin, there are so much specs to follow etc. Do you have any good reading to suggest to just get started with the basics and work from there?

Thanks in advance!

**Comment by HugoGiraudel** [comment]
Hello Viktor!

Interesting you ask; it appears to be quite a usual situation a lot of developers are facing. “I want to build a more accessible and inclusive web, but where do I even start?”

First things first: while fundamentally connected, inclusivity and accessibility on the web are slightly different. Accessibility is about making sure people suffering from a handicap can use the web (be it physical, cognitive, visual or auditive). Inclusivity is about making sure everyone, despite who they are and their characteristics, can feel valued and respected as users and humans.

Accessibility is governed by standards (Section 508, WCAG2A, WCAG2AA…). However, they can feel very overwhelming. I’m going to be honest with you: I never read them fully. Kind of like HTML/CSS specifications, they are mostly dedicated to implementors rather than developers (although it’s still insightful to read). 

As a general rule of thumb, accessibility works until you break it. It’s always good to go simple. ARIA shenanigans can be useful, but if you can avoid them and go simple, probably best to do so. It’s usually best not to sweat accessibility too much, it doesn’t have to be over-engineered, but it’s easy to do so.

It’s good to remember that accessibility is not just about blind users. Yes, it is a large part of it, but not only. Also, handicaps are not always permanent. They can be situational and/or temporary. This graphics from Microsoft’s accessibility manifesto is a good example: cdn-images-1.medium.com/max/1600/1*O_u….

Now when it comes to inclusivity, then there is a lot you can do. Because inclusive design is not something you do correctly or not. It’s something you iterate on to make better little by little. This is where I still find joy and interest in web development. A good way to grasp the depth of it could be to read these feedback from Twitter: hugogiraudel.com/2017/07/02/acc…. I also tweeted a lot about inclusivity and accessibility in this 100-tweets long thread: twitter.com/i/moments/8770….

Again, inclusivity is fun because there is no end to it. You can always do more, you can always do better. I find it nice to take it as an interesting challenge. I try to consider more people. People with a different background, a different life, a different story, different tastes, different expectations. And then, I try to do my best. It’s usually surprising at how little it takes to make it a lot better for some people.

**Comment by brunopulis** [comment]
Hi @HugoGiraudel great explanation, I have a repository that contains some references about a11y, maybe it's useful to help anybody how starting in a11y. :)
